keyboards/ristretto/readme.md

```diff
@@ -0,0 +1,16 @@
+# Ristretto
+
+![Ristretto](https://i.imgur.com/VHX6ktEh.jpg)
+![Ristretto-2](https://i.imgur.com/Hrc8vkph.jpg)
+
+An Integrated, Column Stagger, 40% Keyboard with OLED and Rotary Encoder
+
+* Keyboard Maintainer: [Fruit](https://github.com/Blewis308)
+* Hardware Supported: Ristretto PCB, Atmega32u4
+* Hardware Availability: Through Group Buy - May 2021
+
+Make example for this keyboard (after setting up your build envionment):
```

Comment:
Could you add information on how to enter the bootloader to flash new firmware on the board?